README.md

```diff
@@ -1,5 +1,13 @@
+
+
+ 
+<div align="center">
+<img width="800"  alt="notification-center-912bb96e009fb3a69bafec23bcde00b0" src="https://github.com/winglang/wing/assets/66887028/6023ceab-060e-4b48-be98-00a546e8c724" alt-text="GIF of Wing's Clouds">
```

Comment:
This gif is a little bit grainy, is there any way it could be less compressed?

Reply:
That's a good question, I had to reduce the size so it's under 10 MB. 
I can see if @amielpb can reduce the size but keep it hi-rez.